docs/buildah-commit.md

```diff
@@ -13,6 +13,14 @@ specified, an ID is assigned, but no name is assigned to the image.
 
 ## OPTIONS
 
+**--cert-dir** *path*
+
+Use certificates at *path* (*.crt, *.cert, *.key) to connect to the registry
```

Comment:
Is there a default?

Reply:
It doesn't default and there's some wonkiness going on that I need to talk over with Miloslav about which may be my lack of openssl cli knowledge.   I'm like this as is, but I'm fine with changing this here if you wish.  But it's the same verbiage as in skopeo and we'd probably want to touch that up too.

Reply:
If I'm reading things (https://github.com/containers/image/blob/master/docker/docker_client.go#L228, https://github.com/containers/image/blob/master/docker/docker_client.go#L136, and the two test cases at https://github.com/containers/image/blob/master/docker/reference/reference_test.go#L45) right, what we're setting will lead us to a subdirectory of `/etc/docker/certs.d` named after the host and port (if specified) of the registry which we're contacting.

Reply:
We probably need to change this code to start using /etc/containers/certs.d as well.  But not in the pull request.  I guess I would rather not point out this default in the man page.

Reply:
Ok, sounds good.  I've created Issue #251 so we can track this.